assemblers/org.osbuild.ostree.commit

```diff
@@ -151,6 +152,11 @@ def main(tree, output_dir, options, meta):
                            stdout=sys.stderr,
                            check=True)
 
+        with open(os.path.join(output_dir, "compose.json"), "r") as f:
+            compose = json.load(f)
+
+        api.metadata(compose)
```

Comment:
Should we namespace this, so that more metadata in principle could be appended in the future? I.e, just make a sub dict called `compose`?

Reply:
I thought about it and decided against it because if felt redundant, since all the metadata belong to the compose of the commit and the data is somewhat namespaced via the keys:
```
"assembler": {
    "name": "org.osbuild.ostree.commit",
    "id": "30e50739bea0d287e1f6ea0390a9d2842251eafc738b1791638662ded7178791",
    "options": {
      "ref": "fedora/x86_64/osbuild-demo"
    },
    "success": true,
    "output": "[/usr/lib/tmpfiles.d/journal-nocow.conf:26] Failed to resolve specifier: uninitialized /etc detected, skipping\nAll rules containing unresolvable specifiers will be skipped.\n\n(rpm-ostree compose commit:16): libdnf-WARNING **: 09:51:46.903: Loading \"/etc/dnf/dnf.conf\": IniParser: Can't open file\nCommitting...done\nfedora/x86_64/osbuild-demo => b1e784b6ddc5f8a32bc8399e4836fc2ce5bdc5ec099d6079df6e902adfff26b0\nMetadata Total: 9749\nMetadata Written: 3457\nContent Total: 30225\nContent Written: 26140\nContent Cache Hits: 0\nContent Bytes Written: 1221467040\nInitializing root filesystem\nCopying data\n",
    "metadata": {
      "ref": "fedora/x86_64/osbuild-demo",
      "ostree-n-metadata-total": 9749,
      "ostree-n-metadata-written": 3457,
      "ostree-n-content-total": 30225,
      "ostree-n-content-written": 26140,
      "ostree-n-cache-hits": 0,
      "ostree-content-bytes-written": 1221467040,
      "ostree-commit": "b1e784b6ddc5f8a32bc8399e4836fc2ce5bdc5ec099d6079df6e902adfff26b0",
      "ostree-content-checksum": "77bca094ddc9c32c51fed4a41d04360a72e1ddb5be6d2a3c571c894d24dd7b16",
      "ostree-version": "32.0.1",
      "ostree-timestamp": "2020-08-11T09:53:00Z",
      "rpm-ostree-inputhash": "30e50739bea0d287e1f6ea0390a9d2842251eafc738b1791638662ded7178791"
    }
  }
```

Reply:
My thinking was that (in general) when we pass something through like this, it means that we could not later add more metadata keys if ever we wanted to add anything that does not come from `compose.json`. I agree that this is unlikely, and just an academic concern, so feel free to keep it as is.

Reply:
I indeed used `compse` as a namespace now, tried to separate the `ostree` stuff out but it got to tedious for little gain.